### DISCUSSION TITLE
Retry failed workflow with `ttl`  deleted after initial `secondsAfterFailure` while still running

### DISCUSSION THREAD

**Discussion by manuelbmar** [argoproj/argo-workflows#12630] [Q&A] [answered]
I am retrying a failed worflow that has steps set up with manual approval and ttl secondsAfterFailure .

Workflow failed : 
<img width="485" alt="image" src="https://github.com/argoproj/argo-workflows/assets/128461527/61449106-95ce-4e54-9e17-946866d9c6d4">

I make a call to /retry method in api rest, the failed step has now gone fine and I manage to advance our workflow correctly.

<img width="375" alt="image" src="https://github.com/argoproj/argo-workflows/assets/128461527/6a5635dc-13b1-48bb-b215-814ab16ffcf3">

The workflow is now waiting for the next step with manual approval as you can see in the screenshot .

But once the TTL secondsAfterFailure is completed, "workflow gone" message is displayed and the workflow status shown in the UI is the one before the /retry action. (screenshot 1).

I think Ttl strategy is not taking into account that the workflow is running, as it has a "suspend" step waiting for approval.

Should I open a bug with this behaviour ?

Thanks

**Comment by shuangkun** [comment]
Can you paste a workflow to help reproduce it？

**Comment by agilgur5** [comment, marked as answer]
Duplicated in issue #12636 as a proper bug report and now has significantly more details
